linera-client/src/client_options.rs

```diff
@@ -434,6 +434,9 @@ pub enum ClientCommand {
         chain_id: Option<ChainId>,
     },
 
+    /// Show the list of BlobIds of the validator
```

Comment:
The local node (not necessarily a validator)

Reply:
Ok.